app/views/browse/_titles.html.erb

```diff
@@ -12,7 +12,7 @@
   <tbody>
     <% title_list.entries.map do |item| %>
       <tr>
-        <td><%= link_to item.value, search_catalog_path('search_field' => 'title', 'q' => item.value) %></td>
+        <td><%= link_to item.value, search_catalog_path("f[#{title_list.field}][]" => item.value) %></td>
```

Comment:
Do we need `#{title_list.field}` here? We only need title_facet, right?

Reply:
yep, I can change it